addons/sys_zeus/initSettings.sqf

```diff
@@ -13,11 +13,11 @@ private _zeusCategory = format ["ACRE2 %1", localize "str_a3_cfghints_curator_cu
     ]
 ] call CBA_fnc_addSetting;
 
-// Ability to hear through the Zeus camera
+// Ability to communicate through the Zeus camera
 [
-    QGVAR(zeusListenViaCamera),
+    QGVAR(zeusCommunicateViaCamera),
```

Comment:
Note: this will reset the setting for everyone.

Reply:
I think it is a significant enough change.

The last setting didn't work as some people assumed it would. In this case since this new setting is more clear I don't think having it reset is a huge issue.

Reply:
We still have to note it in release.